iconvertor.py

```diff
@@ -0,0 +1,11 @@
+from abc import ABC, abstractmethod
+from typing import List, NoReturn
+
+from card import Card
+
+
+class IConvertor(ABC):
```

Comment:
Is this what you said that implements interface in Python or is this just an abstract class? 
I trust you this is the best practice in this case :)

Reply:
:D